lightwood/encoders/numeric/ts_numeric.py

```diff
@@ -48,23 +48,18 @@ def encode(self, data, extra_data=None):
                     mean = self._abs_mean
                 if real is not None and mean > 0:
                     vector[0] = 1 if real < 0 and not self.positive_domain else 0
-                    vector[1] = math.log(abs(real)) if abs(real) > 0 else -20
-                    vector[2] = real / mean
+                    vector[1] = real / mean
                 else:
                     log.debug(f'Can\'t encode target value: {real}')
 
             else:
-                vector = [0] * 4
```

Comment:
im very confused why the dimensionality changes? My best guess is that numeric has 3 terms and time series may have 2?

Reply:
Same as above: the log term was not being used anyway, so I removed it. In this case (non-target column) the first element is there following the standard numerical encoder, as far as I can remember it's there as a sanity check (i.e. it should always decode close to 1, else something is wrong and we decode `None`).